x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/styled_components.tsx

```diff
@@ -9,7 +9,7 @@ import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 
 export const StyledEuiFlexGridGroup = styled(EuiFlexGroup)`
   display: grid;
-  grid-template-columns: 25% 45% 30%;
+  grid-template-columns: 33% 45% 22%;
```

Comment:
Make sure this card is rendered correctly in Fleet under Edit Integration Policy, where we currently show the Trusted app card when displaying our Policy form.

Reply:
I checked and it does look fine 
![image](https://user-images.githubusercontent.com/227916/142486770-567e162c-beb0-4780-ba56-fff966690a49.png)